extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -173,11 +177,13 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 		});
 
 		assessmentResults?.items.forEach((item) => {
-			const dbIndex = serverDatabases.indexOf(item.databaseName);
-			if (dbIndex === -1) {
-				serverLevelAssessments.push(item);
-			} else {
-				databaseLevelAssessments[dbIndex].issues.push(item);
+			if (item.appliesToMigrationTargetPlatform === 'AzureSqlManagedInstance') {
+				const dbIndex = serverDatabases.indexOf(item.databaseName);
```

Comment:
Could you explain what this is doing? I'm a little confused why a database name not existing in the list of databases would be considered a "server level assessment"

Reply:
So all the server level issues are returned with an empty database string. I am using that to separate out server level assessments from database assessments.

Reply:
That's unfortunate - there's no other way to tell what are the server level issues? Guess an empty DB name isn't valid anyways but still not the cleanest. But if that's what it is then that's what it is.

Reply:
We can probably modify the object returned by mssql to have this information. Will do that after PP.